Kuroba/app/src/main/java/com/github/adamantcheese/chan/ui/view/MultiImageView.java

```diff
@@ -636,6 +644,13 @@ public void onAudioSessionId(int audioSessionId) {
     }
 
     private void setOther(Loadable loadable, PostImage image) {
+        BackgroundUtils.ensureMainThread();
+
+        if (getWidth() == 0 || getHeight() == 0) {
```

Comment:
This will necessarily never be the case, as these properties are ensured to be greater than zero by waitForMeasure.

Reply:
The same as above.